### ISSUE TITLE
There is an error on installing Clover

### ISSUE THREAD

**Issue by yuningchina** [corpnewt/gibMacOS#51]
Gathering info...
 - Checking https://api.github.com/repos/CloverHackyColor/CloverBootloader/releases
 - Got CloverISO-5099.tar.lzma
Downloading...
Downloaded 2.47 MB of 2.47 MB (100.00%)
Extracting CloverISO-5099.tar.lzma...
Extracting CloverISO-5099.tar...
Extracting EFI from Clover-v2.5k-5099-X64.iso...
Extracting boot0af from Clover-v2.5k-5099-X64.iso...
Extracting boot1f32alt from Clover-v2.5k-5099-X64.iso...
Extracting boot6 from Clover-v2.5k-5099-X64.iso...
No partitions located on disk!

**Comment by corpnewt** [comment]
Yeah - most issues can be fixed by doing one or more of the following:

* Updating Windows
* Trying another USB drive
* Removing any 3rd party partition or disk management software
* Trying on another computer

-CorpNewt
